ecs/src/nox/ecs/EntityManager.cpp

```diff
@@ -33,6 +36,196 @@ nox::ecs::EntityManager::registerComponent(const MetaInformation& info)
 void
 nox::ecs::EntityManager::configureComponents()
 {
+    auto typeIdentifierComp = [](const TypeIdentifier& lhs, const TypeIdentifier& rhs)
```

Comment:
Would be nice to remove this lambda, because now we have to structures that are used exactly the same. Which is probably not needed.